lib/maverick/api.ex

```diff
@@ -47,6 +47,7 @@ defmodule Maverick.Api do
       @otp_app Keyword.fetch!(opts, :otp_app)
       @root_scope opts |> Keyword.get(:root_scope, "/")
       @router Module.concat(__MODULE__, Router)
+      @modules opts |> Keyword.get(:modules, [])
```

Comment:
i don't know why i have the opts inside the parens on line 47 but i'm piping them in on line 48; i'd just get rid of all of those single-pipes, no idea what i was thinking